server_admin/topics/authentication/webauthn.adoc

```diff
@@ -43,6 +38,8 @@ If you want to have WebAuthn required for all users:
 
 image:images/webauthn-browser-flow-required.png[]
 
+* In the `Bindings` menu, change the browser flow to `WebAuthn Browser`
+
```

Comment:
The _* Set the alias to "Conditional 2FA"_ sentence below is missing 'Click "Save" button" part' (it's not a problem of this PR though, looks the change was introduced before).

Right now it looks like:
![conditional_2fa_no_save](https://user-images.githubusercontent.com/8414918/72801791-9566a500-3c4a-11ea-9a98-b03fd9fda6fb.png)

But would be good (IMHO) if it would read as follows:
_Set the alias to "Conditional 2FA" and click on `Save`_

Reply:
Yet (from the testing), the _* Select `Condition - User Configured` using the drop down and click on "Save"_ sentence below is missing the sentence, that the requirement of the _Select `Condition - User Configured` execution should be set to `Required` (defaults to `Disabled`). Thus the proposal is to change the sentences as follows (the following image displays how it looks currently):

![user_configured](https://user-images.githubusercontent.com/8414918/72802206-8c2a0800-3c4b-11ea-99eb-6862be71a353.png)

The change would be to add the sentence _Set the Requirement of `Condition - User Configured` execution to Required` so the flow part looks like this:

![webauthn_form_webauthn_only_if_registered](https://user-images.githubusercontent.com/8414918/72802385-f642ad00-3c4b-11ea-8581-7f07275acefa.png)

because without changing the condition of `Condition - User Configured` from `Disabled` (the default) to `Required` the user is not asked to enter / login via WebAuthn even when they have WebAuthn already configured.

Reply:
Thanks, I will apply both suggestions from your comment above